lms/static/js/ajax-error.js

```diff
@@ -1,7 +1,7 @@
 $(document).ajaxError(function(event, jXHR) {
     if (jXHR.status === 403 && jXHR.responseText === 'Unauthenticated') {
         var message = gettext(
-            'You have been logged out of your edX account. ' +
+            'You have been logged out of your account. ' +
```

Comment:
Can we sub in siteName here? (I'm not sure how that would work in JS)

Reply:
Thanks @sarina. It's a bare JS file that receives no context from LMS.

I wasn't sure if we have those variables in global scope, do we?

Reply:
No idea... 🤔 ... maybe @davidjoy knows!

Reply:
We have some user meta data but no platform metadata:

```html
<script type="application/json" id="user-metadata"> {"username": "OmarIthawi", ...}</script>
```

Reply:
I don't know offhand, but trust @OmarIthawi 's assessment that it'd be onerous to get the platform/site name into this file.  

The closest I see is in some React code for student account deletion where we're setting `siteName` and `platformName` so we can use them later in `gettext` inside the react component.  It's a bit of a weird hack which I don't think we should reproduce.

I'd say just merge it, since this is a relatively uncommon case and is probably pretty clear anyway.

References for the React/window thing:

https://github.com/edx/edx-platform/blob/master/lms/templates/student_account/account_settings.html#L83
https://github.com/edx/edx-platform/blob/master/lms/static/js/student_account/StudentAccountDeletionInitializer.js#L25
https://github.com/edx/edx-platform/blob/master/lms/static/js/student_account/components/StudentAccountDeletion.jsx#L92

Reply:
OK! Thanks both!